### DISCUSSION TITLE
Select text via Ctrl+Shify+<-/->

### DISCUSSION THREAD

**Discussion by valerii15298** [nushell/nushell#9071] [Q&A]
Coming from *Powershell* I am used to selecting text in the current prompt via `Ctrl+Shify+<-/->` but I cannot do that in **Nushell**, is there any alternative to selecting text via keyboard without mouse in **Nushell**?

**Comment by fdncred** [comment]
Not that I know of, although if the option exists in crossterm, it may be able to be implemented in reedline.
